### DISCUSSION TITLE
Does Neo Store do unattended updates?

### DISCUSSION THREAD

**Discussion by matchboxbananasynergy** [NeoApplications/Neo-Store#254] [Q&A]
With the recent split between Neo Store and Droid-ify, I'm trying to find what the meaningful differences between the two apps are.

One of the most meaningful differences is unattended updates. What is the status on that for Neo Store right now?

**Comment by WofWca** [comment]
Seems to be the case (but only when the app is in the foreground??)

https://github.com/NeoApplications/Neo-Store/issues/20
https://github.com/NeoApplications/Neo-Store/issues/208
https://github.com/NeoApplications/Neo-Store/pull/159

**Comment by machiav3lli** [comment]
Fully fixed in 1.0.0
